### PR TITLE
Revise and complete Lao translations

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
We've now received the majority of the materials for a Lao translation. In this PR, I'm adding the translation files that were previously missing and revising lo.json to reflect the updated translations.

### CODE REVIEW

**Comment by Abby-Wheelis** [comment]
In comparing to the es folder, I'm missing the `intro` folder with 3 .html files in it... I can add the English versions here but don't see a place that I have the Lao versions of the text on those pages, particularly the `consent-text`

**Comment by Abby-Wheelis** [comment]
I have been trying to test the new translations in the dev app, but haven't been able to figure it out. I've worked with the instructions [here](https://github.com/e-mission/e-mission-docs/blob/master/docs/dev/front/how_to_add_a_language.md) and have tried on a branch off of framework-migration-prep and one from master-for-platform (of e-mission-phone) since the translations work a little differently. I've seen translations from the config file, and everything works relatively smoothly, but I'm still struggling with seeing my translations on the dev app. Does anyone have any recommendations?

**Review by shankari — APPROVED** [review]

**Comment by shankari** [comment]
> In comparing to the es folder, I'm missing the intro folder with 3 .html files in it... I can add the English versions here but don't see a place that I have the Lao versions of the text on those pages, particularly the consent-text

Ah yes! Let me see if I have that in any of the emails from Kosol, but I don't think we do. However, note that in `master_for_platform`:
(1) the summary text is filled in from the dynamic config anyway, and
(2) most of the consent-text is duplicated from the join page. It might be a bit time-consuming, but can you try your best to fill in the consent-text with the corresponding values from the join page template? I can ask Kosol for a proper translation after the demo is complete.

**Comment by shankari** [comment]
> I've seen translations from the config file, and everything works relatively smoothly, but I'm still struggling with seeing my translations on the dev app.

If you recall, the translations are downloaded into `locale` (in `hooks/before_prepare/download_translation.js`) and are then copied into the `i18n` folder (in `hooks/before_build/android/android_copy_locales.js` and `hooks/after_platform_add/ios/ios_copy_locales.js`). You should check to make sure that all of that is happening and the `lo` translations are in the `i18n` folder.

If all that is correct, check whether you have enabled `lo` here: 
https://github.com/e-mission/e-mission-phone/blob/60694a9bdcd60044273f06944e743bda3fd5ab0b/www/js/i18nextInit.ts#L12

**Comment by Abby-Wheelis** [comment]
Thank you for pointing those things out, I was able to see most of the translations load through, I'm pretty sure it was the enabling of lo that I was hung up on, I kept getting confused with different branches. 

I'll work on roughly mapping in those missing files, there some other places that ended up pretty rough (like the join section of lo.json) so those can get smoothed out post demo!

**Comment by Abby-Wheelis** [comment]
> (2) most of the consent-text is duplicated from the join page. It might be a bit time-consuming, but can you try your best to fill in the consent-text with the corresponding values from the join page template? I can ask Kosol for a proper translation after the demo is complete.

I've done some of this, but it's definitely not perfect yet. I've carried over what I thought I could, and there's some left in English, I can keep trying to take the English out, but I'm not sure to what extent we want to preserve content vs "brute force" a full translation right now. I've spot checked with google translate Lao -> English, so I could translate the remaining sections that way and probably get relatively close? I'm not willing to put full faith in Google Translate but it might get us close enough that someone familiar with the project and the language could review

**Comment by shankari** [comment]
> I'm not willing to put full faith in Google Translate but it might get us close enough that someone familiar with the project and the language could review

Yeah let's do that. I will ask for this final translation (basically `consent-text-lo.html`) when I tell Kosol that everything else is done.

File: lo/intro/summary-lo.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	<div class="intro-space"></div>
+	<li> ✔️  {{template_text.summary_line_2}}
+	<div class="intro-space"></div>
+	<li> ✔️  {{template_text.summary_line_3}}
+	</ol>
+	</div>
+</ion-content>
+<ion-footer-bar class="no-bgColor" id="intro-footer">
+  <button class="button button-block button-outline button-positive" ng-click="next()">
+    EMPEZANDO
```
Comment (shankari) [inline lo/intro/summary-lo.html:21]: this needs to be changed

File: lo/intro/consent-text-lo.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    ລະບົບນີ້ຈະນໍາໃຊ້ ເພື່ອເປັນການເກັບກໍາຂໍ້ມູນໃນການຕິດຕາມ ການເດີນທາງ ເພື່ອໃຫ້ເຂົ້າເຖີງຮູບແບບການເດີນທາງຂອງການນໍາໃຊ້ພາຫະນະ.
+    </p><p>
+    ມັນຍັງຈະຂໍໃຫ້ເຈົ້າບັນທຶກການເດີນທາງທີ່ມີຄວາມຮູ້ສຶກເຫຼົ່ານີ້ເປັນໄລຍະໆດ້ວຍ semantic
+    ປ້າຍຊື່, ເຊັ່ນ: ຮູບແບບການເດີນທາງ, ຈຸດປະສົງ, ແລະຮູບແບບການທົດແທນ.
+    </p><p>
+    It will also request sociodemographic information such as your approximate age,
+    gender, and household type. The sociodemographic factors can be used to
+    understand the influence of lifestyle on travel behavior, as well as generalize
+    the results to a broader population.
+    </p><p>
```
Comment (shankari) [inline lo/intro/consent-text-lo.html:40]: Google wasn't good enough?

**Review by shankari — CHANGES_REQUESTED** [review]

File: lo/intro/consent-text-lo.html (hunk 2 above)
Comment (Abby-Wheelis) [reply lo/intro/consent-text-lo.html]: I accidentally got two lines confused earlier, thanks for pointing that out, it's fixed now!

File: lo/intro/summary-lo.html (hunk 1 above)
Comment (Abby-Wheelis) [reply lo/intro/summary-lo.html]: I've updated this now!

**Review by shankari — APPROVED** [review]

**Comment by shankari** [comment]
There's some early churn here, so I am going to squash merge again.
But the PRs are getting better and soon, I won't have to do it any more!

**Pull request merged**